src/octoprint/plugins/action_command_notification/__init__.py

```diff
@@ -120,6 +121,12 @@ def action_command_handler(self, comm, line, action, *args, **kwargs):
             return
 
         message = parameter.strip()
+
+        filters = self._settings.get(["filters"]).split("\n")
```

Comment:
For performance, would it be better to pre-fetch the filters and set it as an attribute on the class? Like `self.filters` or something simple. It would save looking them up in the settings for every line that is received. Probably only a small difference, but the less work that is done on every line then the less likely stuttering issues due to slow processing will be?

Reply:
To take it further (since the `re.fullmatch` below will have to do the work of compiling the regex for every single line too, which is a big red flag): fetch once on start (and after settings save), compile into regex (`re.compile`), us THAT on every received `action:notification` line.